### PR TITLE
remove depwarns and test warnings

### PR DESCRIPTION
**Pull request by SimonDanisch**
will let travis test 0.4 ;)

### CODE REVIEW

**Comment by timholy** [comment]
With apologies for piling on to your PR, this is now passing on julia-0.4 as well. But, I decided to drop continued support for julia-0.3. Is this OK?

**Comment by SimonDanisch** [comment]
That's all very much okay! :)

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,6 @@ os:
   - linux
   - osx
 julia:
-  - 0.3
```
Comment (tkelman) [inline .travis.yml]: appveyor too

**Comment by tkelman** [comment]
"don't display warnings in tests" strikes me as a very, very bad idea

File: .travis.yml (hunk 1 above)
Comment (timholy) [inline .travis.yml]: fixed

**Pull request merged**